pmd-apex-jorje/pom.xml

```diff
@@ -81,7 +81,7 @@
     <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>26.0-jre</version>
+        <version>26.0-android</version>
```

Comment:
why android?

Reply:
For pmd-core, the JRE variant of Guava cannot be used because it requires Java 8. The Android variant, despite the name, only means that JRE7 can be targeted.

In order to avoid clashes between pmd-core and pmd-apex-jorje, I also adjusted the latter to use the Android variant. Once PMD targets JRE8, this can be reverted.

I believe the JRE and Android variants should be mostly equivalent.

Reply:
I've reverted this change. It shouldn't be necessary anymore, since pmd-core is no longer using Guava.